homeassistant/components/sensor/__init__.py

```diff
@@ -189,6 +190,9 @@ class SensorDeviceClass(StrEnum):
     # volume (VOLUME_*)
     VOLUME = "volume"
 
+    # weight (g, kg, mg, µg, oz, lb)
```

Comment:
Can we make it more clear in the docstring that it's mass, but we name it weight because it fits better with every day language?